css/BadgeOverlaySupport.less

```diff
@@ -0,0 +1,17 @@
+.moon-badge-overlay-support .moon-image-client {
+	color: white;
+	padding: 0;
+	background-color: rgba(0, 0, 0, 0.5);
+	.moon-icon {
+		width: 36px;
+		height: 36px;
+	}
+}
+.moon-badge-overlay-support .moon-image {
+	display: block;
+	margin: 0;
+}
+.moon-badge-overlay-support.top .moon-image-client {
+	bottom: initial;
+	top: 0;
+}
```

Comment:
Lets rearrange this file like this:
```
.moon-badge-overlay-support {
	.moon-image-client {
		color: #fff;
		padding: 0;
		background-color: rgba(0, 0, 0, 0.5);

		.moon-icon {
			width: 36px;
			height: 36px;
			line-height: 36px;
		}
	}
	&.top .moon-image-client {
		bottom: initial;
		top: 0;
	}
	.moon-image {
		display: block;
		margin: 0;
	}
}
```
It will visually group the rules and the CSS will be just the same. (changed the color from "white" to "#fff" for consistent color naming across the framework, added `line-height`, and swapped the order of the `.top` class.